templates/base/paginate.tmpl

```diff
@@ -15,7 +15,7 @@
 					{{if eq .Num -1}}
 						<a class="disabled item">...</a>
 					{{else}}
-						<a class="{{if .IsCurrent}}active{{end}} item" {{if not .IsCurrent}}href="{{$.Link}}?page={{.Num}}{{if $paginationLink}}&{{$paginationLink}}{{end}}"{{end}}>{{.Num}}</a>
+						<a class="{{if .IsCurrent}}active{{end}} item content-center" {{if not .IsCurrent}}href="{{$.Link}}?page={{.Num}}{{if $paginationLink}}&{{$paginationLink}}{{end}}"{{end}}>{{.Num}}</a>
```

Comment:
```suggestion
						<a class="{{if .IsCurrent}}active {{end}}item content-center"{{if not .IsCurrent}} href="{{$.Link}}?page={{.Num}}{{if $paginationLink}}&{{$paginationLink}}{{end}}"{{end}}>{{.Num}}</a>
```

Reply:
This is something I want to address together in another PR.

Reply:
Let's not do this here but in https://github.com/go-gitea/gitea/pull/21855.